java-rx-smgrsdk/src/main/java/io/vantiq/svcconnector/StorageManagerVerticle.java

```diff
@@ -90,6 +90,16 @@ private Flowable<?> dispatch(SvcConnSvrMessage msg) {
                             + msg.procName);
                 }
                 String procName = parts[parts.length - 1];
+                
+                // In the future, it should be set by an explicit value isSystemRequest. For compatibility reasons, we
+                // must accept appending "__system" to the requestId as equivalent
```

Comment:
The previous Java message processing throws an error and never responds if there's an unexpected field in the message. The new version won't do that, but we still need a way to bypass the issue so that 1.40 is still compatible with existing storage managers.